### ISSUE TITLE
Add cookie problem with Bstation/Bilibili

### ISSUE THREAD

**Issue by erickmufath** [JunkFood02/Seal#1466]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

This maybe also part of 'Feature request'.

So when you want to add cookies in the settings by going to the web and logging in, Bstation will automatically redirect to the application, this is the problem. I could use a username and password but it often doesn't work because it requires verification, so I want to use cookies instead. Therefore, I want you to fix this. Opening a Bstation website in Chrome will also redirect you directly to the application, unless you use desktop mode, but compared to Chrome, Seal doesn't redirect it to the application with just a website error message in general.

Basically I want to make a browser on Seal so that at any time the web can be loaded in desktop mode to pass some web like Bstation, perhaps by adding a desktop browser user-agent option or a toggle enable desktop mode button like the one on Chrome Mobile before entering the browser display.

### To Reproduce

1. Open seal and click Settings icon in left top screen
2. Click network
3. Click cookie
4. Click create new cookie
5. Paste https://www.bilibili.tv/id and click new cookie again below url box

### Error reports

Web page not found

### Screenshots & Screen Records

![Screenshot_20240419-163100_Seal](https://github.com/JunkFood02/Seal/assets/90117993/536668e7-9044-4979-85cc-17572e466f5b)

**Comment by JunkFood02** [comment]
You may need to update to v1.12.1 and try again
